music21/abcFormat/translate.py

```diff
@@ -279,6 +278,7 @@ def parseTokens(mh, dst, p, useMeasures):
             #ql += t.quarterLength
 
         elif isinstance(t, abcFormat.ABCNote):
+            pit = t.pitchName
```

Comment:
revert please -- this change takes something that is explicit (t.pitchName) into something that I need to look up where `pit` comes from

Reply:
Roger that.